src/sentry/receivers/onboarding.py

```diff
@@ -255,7 +255,8 @@ def record_first_replay(project, **kwargs):
             platform=project.platform,
         )
         logger.info("record_first_replay_analytics_end")
-        try_mark_onboarding_complete(project.organization_id)
+        user: RpcUser = project.organization.get_default_owner()
```

Comment:
This feature flag won't really work as you want it. `get_default_owner` doesn't return user who performed the aciton, but the default owner of the organization. 

Maybe it is better if you switch this feature flag to org level immediately - we can test it in telex organization, than to try to use it like this

Reply:
it will work with the organizations i create with my email - I can create up to 10 per day... the problem with restricting the feature flag to one organization is that i can't reset the quick start without updating the tables and it's annoying to always add an organization slug to the Sentry options automation

Reply:
The hard thing about this is that the checks for the quickstart act on all kinds of tables if I understood correctly, so there is no way of "resetting" an org efficiently at the moment.